protos/proto/aptos/transaction/v1/transaction.proto

```diff
@@ -403,6 +403,7 @@ message Signature {
     TYPE_MULTI_ED25519 = 2;
     TYPE_MULTI_AGENT = 3;
     TYPE_FEE_PAYER = 4;
+    TYPE_SECP256K1_ECDSA = 5;
```

Comment:
Two things:
1. Are we keeping TYPE_SECP256K1_ECDSA and friends or TYPE_SINGLE_SENDER and friends?
2. Unless we're confident we haven't rolled these out yet, instead of totally deleting one of them we should mark it as reserved.

Reply:
Let's do the `reserved`; we unintentionally released the proto change to testnet and mainnet.

Reply:
Which one are we not using? TYPE_SECP256K1_ECDSA or TYPE_SINGLE_SENDER? I'll mark it as reserved.

Reply:
I realized it'll be challenging to handle the oneof removal.. let's remove instead of `reserved`; i expect no clients expect these fields.

Reply:
Spoke with Maayan, we're getting rid of TYPE_SECP256K1_ECDSA.